src/app-pages/map/components/location-stream-controls/LocationStreamControls.js

```diff
@@ -31,6 +34,9 @@ const LocationStreamControls = ({ locationDetailData, fullScreen, doLocationsMap
     justifyContent: "space-between",
   };
 
+  // Don't show stream controls if there were no stream locations found
+  if( options.length === 0 ) return <></>
```

Comment:
If streamLocations is an empty array, then the "Back to map" link doesn't display on the location page